templates/shared/_cra-banner.html

```diff
@@ -2,10 +2,10 @@
   <div class="row">
     <div class="p-notification--information">
       <div class="p-notification__content">
-        <div class="p-notification__message">Need information about the CRA? Canonical is committed to delivering Cyber Resilience Act (CRA) compliant Ubuntu. To learn more, 
-          <a href="/security/contact-us">contact our sales team.</a>
+        <div class="p-notification__message">
+          Need information about the Cyber Resilience Act (CRA)? Canonical is committed to delivering CRA-compliant Ubuntu. To learn more, visit our dedicated webpage for the understanding the CRA and its requirements, or <a href="/security/contact-us">contact our sales team</a>.
```

Comment:
```suggestion
          Need information about the Cyber Resilience Act (CRA)? Canonical is committed to delivering CRA-compliant Ubuntu. To learn more, visit our dedicated webpage for understanding the CRA and its requirements, or <a href="/security/contact-us">contact our sales team</a>.
```
this extra "the" is likely a typo in the copydoc.